design/overview.md

```diff
@@ -81,43 +82,36 @@ And provide a security equivalent to or greater than IBC.
 
 ## Bootstraping
 
-This is based on [Installing Peggy on a live cosmos chain](notes.md#installing-peggy-on-a-live-cosmos-chain).
-We also assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
-as approaches to that are discussed in many other places. Here we focus on the *activation* step.
+We assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
+as approaches to that are discussed in many other places. Here we focus on the _activation_ step.
 
-Set up and register orchestrators:
+1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
+1. Each `Operator` configures their `Orchestrator` with the `Peggy Bridge ID` and `start threshold` they wish, their `Orchestrator` is to reject any proposals that do not agree on these values.
+1. Anyone submits a Peggy contract using a known codehash and the current validator set of the Cosmos zone to an Ethereum compatible blockchain. Note that step #1 must happen first because the `Orchestrator` Ethereum Addresses must be made known in order for verification to be possible.
+1. Each `Operator` may or may not configure their `Orchestrator` with the above Peggy contract address
+1. If configured with an address the `Orchestrator` checks the provided address. If the contract passes validation the `Orchestrator` signs and submits the contract address. Validation is defined as finding the expected code hash and a validator set matching the current set within `Allowed validator set delta`. This message also includes the proposed `Peggy Bridge ID` and `start threshold`
+1. A contract address is considered adopted when the proposed `start threshold`. Because validator sets change quickly `Orchestrators` not configured with a contract address observe the Cosmos blockchain for submissions. When an address is submitted they validate it and approve it themselves if it passes. This results in a workflow where once a valid contract is proposed it will be ratified in a matter of a few seconds.
```

Comment:
Since ever message above may have a different `start threshold`, this can become very confusing to compare.

If you want everyone to define different ones, please add an section explaining how we make this start threshold check? What if I have 5% of the stake and submit a 5% start threshold message, does it start automatically?

I think easiest is to define the threshold as an _objective_ parameter on chain (which can be updated by governance)

Reply:
I guess we can require full governance votes on every parameter before anyone is allowed to submit a contract. It will be a sort of lengthy process with lots of human intervention but it doesn't affect us that much.